src/main/java/seedu/address/model/person/GraduationDate.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the Graduation Date of a Person in the address book, as combination of a Month and Year
+ * Guarantees: immutable; is valid as declared in {@link #isValidGraduationDate(String)}
+ */
+public class GraduationDate {
+
+    public static final String MESSAGE_CONSTRAINTS = "Graduation Date should be of the format month-year"
```

Comment:
Is it clearer if we say MM-YYYY instead of month-year?

Reply:
Format was meant to be as natural as possible. Added a comment on the date format for clarity